editor.planx.uk/src/pages/FlowEditor/components/forms/FormModal.tsx

```diff
@@ -163,6 +178,12 @@ const FormModal: React.FC<{
               data: any,
               children: Array<any> | undefined = undefined,
             ) => {
+              if (containsMadeLink(data.data)) {
+                toast.error(
+                  "Content errors detected. Please fix before continuing.",
```

Comment:
nit: can this be a more specific error if we know `containsMadeLink()` - eg `Legislation GOV UK links incorrectly ending in "/made" detected in your content. Please fix before continuing.` ?